libraries/resource_providers/api500/synergy/interconnect_provider.rb

```diff
@@ -1,4 +1,4 @@
-# (c) Copyright 2017 Hewlett Packard Enterprise Development LP
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
```

Comment:
we don't need to update the copyright for files which have no updates

Reply:
@soodpr Done